internal/sync/sync.go

```diff
@@ -91,74 +91,70 @@ func (s Sync) syncPaths(delete bool, local, remote string) error {
 func (s Sync) remoteCmd(ctx context.Context, prog string, args ...string) error {
 	conn, err := s.Client.DialWsep(ctx, &s.Env)
 	if err != nil {
-		return err
+		return xerrors.Errorf("dial websocket: %w", err)
 	}
-	defer conn.Close(websocket.CloseNormalClosure, "")
+	defer func() { _ = conn.Close(websocket.CloseNormalClosure, "") }() // Best effort.
 
 	execer := wsep.RemoteExecer(conn)
 	process, err := execer.Start(ctx, wsep.Command{
 		Command: prog,
 		Args:    args,
 	})
 	if err != nil {
-		return err
+		return xerrors.Errorf("exec remote process: %w", err)
 	}
-	go io.Copy(os.Stdout, process.Stderr())
-	go io.Copy(os.Stderr, process.Stdout())
+	// TODO: Consider checking the copy errors.
+	go func() { _, _ = io.Copy(os.Stdout, process.Stdout()) }() // Best effort.
+	go func() { _, _ = io.Copy(os.Stderr, process.Stderr()) }() // Best effort.
 
-	err = process.Wait()
-	if code, ok := err.(wsep.ExitError); ok {
-		return fmt.Errorf("%s exit status: %v", prog, code)
-	}
-	if err != nil {
+	if err = process.Wait(); err != nil {
+		if code, ok := err.(wsep.ExitError); ok {
+			return fmt.Errorf("%s exit status: %d", prog, code)
```

Comment:
we should be consistent, I think about which Errorf questions.

Reply:
I was wondering about it, as it was already there, I opted not to change it as there might have been a reason?

Reply:
I'm not sure, I can't imagine a way that it would matter, and it feels like this is the right kind of PR to make that change. I don't feel particularly strongly at all.